content/resources/known-amendments.md

```diff
@@ -688,6 +688,8 @@ Amendment `fixNFTokenRemint` would change the way NFT sequence numbers are const
 
 The amendment also introduces a new account deletion restriction. An account can only be deleted if `FirstNFTSequence` + `MintedNFTokens` + 256 is less than the current ledger sequence (256 was chosen as a heuristic restriction for account deletion and already exists in the account deletion constraint). Without this restriction, an NFT could still be re-minted under certain conditions.
 
+**Warning:** This is a **breaking change** for projects & tools relying on their own locally computed NFTokenID for minted tokens. As this fix changes the way a TokenID is calculated, without updating local code the wrong NFTokenIDs will be generated locally. A [well known reference implementation in JavaScript](https://gist.github.com/N3TC4T/a20fb528931ed009ebdd708be4938748?permalink_comment_id=4738760#gistcomment-4738760) can easily be patched with backwards compatibillity.
```

Comment:
> As this fix changes the way a TokenID is calculated

This isn't quite accurate.

The `FixNFTokenRemint` amendment indeed alters how the sequence is calculated but **doesn't** affect the `NFTokenID` itself. It's important to note that manually parsing the `NFTokenID` based on the assumption that `Sequence` is generated from the `MintedNFToken` field from `AccountRoot` might not be a best practice. The XLS20 specification doesn't explicitly state that `Sequence = MintedNFToken` but rather describes it as an "_(automatically generated) monotonically increasing 32-bit sequence number._"

Therefore, the `FixNFTokenRemint` amendment should NOT be considered a breaking change. Instead, those who parse the `NFTokenID` by concatenation should refrain from doing so and adopt an [approach](https://github.com/XRPLF/xrpl.js/blob/bff4144fbfb3dd6c0cd2dc3726ef418daea58226/packages/xrpl/src/utils/getNFTokenID.ts#L36) like the one used in xrpl.js, where the NFTokenPages are traversed to determine the difference.

Reply:
@shawnxie999 It's computed differently. Which means that based on implementation (be it preferred or not) doing what you always did no longer works and results in unexpected outcomes. Which means it's breaking.

Non breaking would mean it stays the same.

Breaking means it behaves differently.

Reply:
```suggestion
**Warning:** This is a **breaking change** for projects & tools relying on their own locally computed NFTokenID for minted tokens. If you have code to calculate NFTokenIDs, you must update it to match the new fomula. For an example of how to do so with backwards compatibility, see this [well known reference implementation in JavaScript](https://gist.github.com/N3TC4T/a20fb528931ed009ebdd708be4938748?permalink_comment_id=4738760#gistcomment-4738760).
```